src/integration/pages/onDemandRadioEpisode/canonicalTests.js

```diff
@@ -2,8 +2,10 @@ import {
   runCoreCanonicalTests,
   runCanonicalAnalyticsTests,
 } from '../../common';
+import runCanonicalOnlyEpisodeTests from './canonicalOnlyEpisodeTests';
 
 export default () => {
   runCoreCanonicalTests();
   runCanonicalAnalyticsTests();
+  runCanonicalOnlyEpisodeTests();
```

Comment:
can we just out the contents of `canonicalOnlyEpisodeTests.js` here

Reply:
👍